src/main/java/com/ldtteam/structurize/commands/ScanCommand.java

```diff
@@ -0,0 +1,108 @@
+package com.ldtteam.structurize.commands;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import com.ldtteam.structurize.management.Manager;
+import com.ldtteam.structurize.util.ScanToolOperation;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.block.Block;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.command.WrongUsageException;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentTranslation;
+
+/**
+ * Command for (un)muting a channel for sender
+ */
+public class ScanCommand extends CommandBase
```

Comment:
What is this here for?

Reply:
Since it should definitely not be in minecolonies but there, it's doing nothing now, will fill it in one of next PRs

Reply:
Ahh okay. At least fix the javadoc then and add a TODO

Reply:
There are two already :D javadocs fixed in many places